web/src/scss/components/watches/WatchConfig.scss

```diff
@@ -48,6 +49,9 @@ a:visited {
   width: 250px;
   height: fit-content;
   padding: 10px;
+  top: 60px;
+  position: sticky;
+  max-height: calc(100vh - 190px);
```

Comment:
Since this page has a footer and header, we use the view port height minus the height of the other elements on the page to constrain the max height of the side nav's container 


i tested this in firefox and seems to work. never used `calc` before